sds_data_manager/constructs/instrument_lambdas.py

```diff
@@ -62,12 +62,24 @@ def __init__(
 
         super().__init__(scope, construct_id)
 
+        # Batch Job Inputs
+        stack = Stack.of(self)
+        job_definition_arn = (
+            f"arn:aws:batch:{stack.region}:{stack.account}:job-definition/"
+            f"{batch_resources.job_definition_name}"
+        )
```

Comment:
I think these could be referenced through the objects `batch_resources.job_definition.job_definition_arn` rather than constructing the ARNs yourself.
https://docs.aws.amazon.com/cdk/api/v2/python/aws_cdk.aws_events_targets/BatchJob.html